### ISSUE TITLE
Plugin doesn't work 

### ISSUE THREAD

**Issue by maurice-schleussinger** [hkockerbeck/ttrss-tumblr-gdpr-ua#2]
Hi, so it seems something changed again, this plugin didn't work for me (basically TTRSS can't fetch the feed even with the plugin enabled). 
However, the original plugin got updated a few months ago and works again (for now): https://github.com/GregThib/ttrss-tumblr-gdpr

Thank you for your work!

**Comment by hkockerbeck** [comment]
I just tested the plugin with the feed of an arbitrarily chosen Tumblr blog, and it worked fine. Which feeds are you having problems with? Are there any other unusual factors in your setup that might have an influence?

**Comment by maurice-schleussinger** [comment]
- the server is located in Germany, so definitely affected by the block
- I changed nothing between installing your plugin and the one by GrebThib
- all Tumblr feeds continued to be marked in red (hovering reveals the typical error message)
- with GrebThib's plugin, the feeds continued to be marked red until the next server refresh

However, your plugin didn't work even then I added a new feed (which worked with GrebThib's plugin). I don't know if there is something unusual about my setup (I used MySQL instead of Postgres on a shared host).

I would suggest to close this issue until somebody else notices a problem. Thanks for the follow up. 🙂
